.jenkins/infrastructure/docker/dockerfiles/windows/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Open Enclave SDK contributors.
```

Comment:
I see no issue with keeping the docker windows images reliant on the PowerShell set up script. I would recommend a minor  readme.md here to explain the decision for future developers.